src/user/schema.graphql

```diff
@@ -407,6 +406,10 @@ type Mutation {
 	Deletes User social
 	"""
 	deleteUserSocial(userId: ID!): Social
+	"""
+	Adds Objectives to the ModuleFeedback
+	"""
+	addObjectives(id: ID!): Social
```

Comment:
- Why are we returning a Social model?
- Where is the second argument that is an array of strings?